Overview.src.html

```diff
@@ -1773,6 +1782,76 @@ <h3 id=http-cors-protocol>CORS protocol</h3>
 Access-Control-Allow-Headers     = #<span data-anolis-spec=http>field-name</span></pre>
 
 
+<h4>CORS protocol and credentials</h4>
+
+<!-- non-normative -->
+
+<p>When <span title=concept-request>request</span>'s
+<span title=concept-request-credentials-mode>credentials mode</span> is "<code>include</code>" it
+has an impact on the functioning of <span>CORS protocol</span> other than including
+<span>credentials</span> in the <span title=concept-fetch>fetch</span>.
+
+<div class="example">
+ <p>In the old days, <code data-anolis-spec=xhr>XMLHttpRequest</code> could be used to set
+ <span title=concept-request>request</span>'s
+ <span title=concept-request-credentials-mode>credentials mode</span> to "<code>include</code>":
+
+ <pre>var client = new XMLHttpRequest()
+client.open("GET", "./")
+client.withCredentials = true
+/* &hellip; */</pre>
+
+ <p>Nowadays, <code title>fetch("./", { credentials:"include" }).then(/* &hellip; */)</code>
+ suffices.
+</div>
+
+<p><span title=concept-request>Request</span>'s
+<span title=concept-request-credentials-mode>credentials mode</span> is not necessarily observable
+on the server, only when <span>credentials</span> exist for
+<span title=concept-request>request</span> can it be observed. In such a scenario the
+<span>CORS request</span> would include <span>credentials</span>, but the
+<span>CORS-preflight request</span> would not. It never does.
+
+<p>The server developer therefore needs to decide whether or not resources "tainted" with
+<span>credentials</span> can be shared. Generally speaking, this is rather unsafe and extreme care
```

Comment:
I found this really confusing on the first two reads, because of your use of "resource" -- which server-side means something very different than what you intend. How about:

    ...whether or not responses "tainted" with credentials...

and similar below.